CLAUDE.md

```
@@ -29,43 +36,81 @@ LocalUp은 Spring Boot 3.5.0 기반의 Kotlin 백엔드 애플리케이션으로
-### 로컬 환경 Docker 컨테이너 실행
+### 코드 스타일 검사
 ```bash
-cd docker/local
-docker-compose -p localup_local up -d
+./gradlew ktlintCheck
 ```
 
-## 프로젝트 구조
+## 아키텍처 개요
 
 ### 계층형 아키텍처
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)